README.md

```diff
@@ -281,6 +282,24 @@ puts response
 # => Button::Response()
 ```
 
+### Offers
+
+##### Get Offers
+
+```ruby
+require 'button'
+
+client = Button::Client.new('sk-XXX')
+
+response = client.offers.get_offers({
+    user_id: "some-user-id",
```

Comment:
Is this the Publisher's id of the user?

Reply:
Yes. All parameters are [here](https://developer.usebutton.com/api#offers)